☑️ 주문 수량에 따른 버튼 색상 구현 설정

kangminseoung opened the pull request:
1. OrderSummaryView
 - minusButton 버튼 함수 호출
 
2. OrderSummaryViewCell
 - 주문 수량에 따른 버튼 색상 구현

kangminseoung committed:
✨ [FEAT] : 주문 수량에 따른 버튼 색상 구현 설정

maxminseok reviewed (approved):
-랑 + 버튼에 통일성이 생겼네요 고생하셨습니다!

DoyleHWorks committed:
Merge branch 'develop' into feat-kms

Quaker-Lee reviewed (approved):
LGTM

DoyleHWorks committed:
Update OrderSummaryViewCell.swift

// MARK: - UIButton Extension for Styling
private extension UIButton {
    /// 버튼 스타일 설정
    func setStyle(borderColor: UIColor, backgroundColor: UIColor) {
        self.layer.borderWidth = 1
        self.layer.borderColor = borderColor.cgColor
        self.layer.cornerRadius = 4
        self.titleLabel?.font = .systemFont(ofSize: 10)
        self.backgroundColor = backgroundColor
        self.setTitleColor(.black, for: .normal)
    }
}

삭제
